spec.html

```diff
@@ -35920,7 +35928,7 @@ <h1>
         <!-- CharacterClassEscape -->
         <emu-grammar>CharacterClassEscape :: `d`</emu-grammar>
         <emu-alg>
-          1. Return the ten-element CharSet containing the characters `0` through `9` inclusive.
+          1. Return the ten-element CharSet containing the characters in the inclusive interval from `0` to `9`.
```

Comment:
Thought we weren't going to do ranges over characters, per https://github.com/tc39/ecma262/pull/2848#discussion_r946283258?

Reply:
True. Fixed by mirroring the steps for `\s` but with `DecimalDigit`. See https://github.com/tc39/ecma262/commit/4bc999dfba270cc8b08b278cff77dd65ebff56a9.

Reply:
I would prefer to keep what was originally there; this change adds a layer of indirection for no benefit.

Reply:
I think it's an improvement over the hand-waving that was there previously. The other way I could see fixing this is listing all the digits inline. But this seems fine, since it's just like what we do already for the other character class escapes.

Reply:
I don't think it's an improvement over the hand-waving that was previously there, which was at no risk of causing confusion and did not require chasing the definition of a nonterminal to parse. Either way, that change doesn't belong in this PR; you can back it out and make a separate PR which we can discuss separately, if you'd like, but I don't want it to go in as part of this PR.

Reply:
This PR changes all current definitions of _numeric_ intervals/ranges, which this isn't.

Reply:
#2876